Toolbar buttons fall outside view-poort

n9iels opened the pull request:
## The bug

On some phones, the end of the toolbar buttons fall of the screen. (Screenshots are from a nexus 5).
This problem occurs on LTR and RTL css.

![screenshot_2015-03-12-18-16-59](https://cloud.githubusercontent.com/assets/2659866/6623620/50505b16-c8e4-11e4-952e-7e4f100b9b04.png)
## How to test this patch

You can test this patch on a mobile phone, but it also can be test by using the developer tool of Google Chrome. Press F12, and click on the mobile icon next to the search icon. Select as device: Nexus 5
1. Go to a page with Toolbar, this can be every view. Langue manager for example
2. Confirm the bug on Nexus 5, also confirm the buttons are not 100% page width on iPhone 5.
3. Apply the patch
4. Confirm the buttons are now shown correctly
5. Change device to iPhone 5 and confirm buttons are know shown the same as nexus 5
6. Switch to a RTL language (Persion for example)
7. Test the patch again
## Note

When compiling the less files, some j-sidebar css compiled too. I didn't make changes to that css... I have no idea if that can cause problems.

Commit:
Change css of toolbar button

fruppel commented:
@test: successful

Issue confirmed and resolved with your patch :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6407">issues.joomla.org/joomla-cms/6407</a>.</sub>

810 commented on administrator/templates/isis/css/template-rtl.css:
```diff
@@ -7526,12 +7526,12 @@ html[dir=rtl] .quick-icons .nav-list [class*=" icon-"] {
-		margin-left: 0;
+		margin-left: 10px;
+		margin-right: 10px;
 		margin-bottom: 5px;
```
please change this to 
````margin: 0 10px 5px 10px; ````
For better codestyle, also do this for less

Commit:
CS + iPhone bug fix

n9iels commented:
Thanks! I changed it :-)

I also noticed a little bug on iPhone size, the buttons were not 100% page width. I solved that one too and changed my test instructions

Erftralle commented:
@test: successfully tested with Nexus 5 emulation on Google Chrome, bug could not be reproduced with iPhone 5 emulation, thanks!

n9iels commented:
On iPhone the problem is that the buttons are not 100% page width. This patch makes the layout for the toolbar the same for al mobile devices. So if the buttons are shown correctly it is perfect :+1:

adhocgraFX commented:
@test testet successfully in chrome with developer tools english and arabic

n9iels commented:
Thanks for testing!

aruttkamp commented:
tested - ok

brianteeman commented:
Setting RTc - thanks for testing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6407">issues.joomla.org/joomla-cms/6407</a>.</sub>

MyDadisDead commented:
Tried it on my Samsung Galaxy 3 and could not replicate the error. Buttons OK

a-fusion commented:
Patch works fine!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6407">issues.joomla.org/joomla-cms/6407</a>.</sub>

phproberto commented:
Merged. Thanks!
